tests/gnrc_sock_dns/README.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+This folder contains a test application for RIOT's sock-based DNS client.
+
+# How to test with native
+
+Setup up a tap interface:
+
+    # sudo ip tuntap add dev riot0 mode tap user $(id -u -n)
```

Comment:
Can you name it `tap0` instead of `riot0`? `native` chooses `tap0` as the default port, so this would make it easier (and more in line with the rest of the doc) for you to "port" this How-To to other platforms than `native` ;-)